compiler/ir.py

```diff
@@ -1192,7 +1193,7 @@ def introduce_aggregators_for_consec_chunks(
                     fileIdGen,
                 )
             else:
-                raise Exception("aggregator kind not yet implemented")
```

Comment:
Konstantinos (from old branch): Maybe add more information in this message. For example, which command does this refer to?